src/Compilers/CSharp/Portable/Symbols/Attributes/WellKnownAttributeData/ParameterWellKnownAttributeData.cs

```diff
@@ -158,5 +158,37 @@ public void AddNotNullIfParameterNotNull(string parameterName)
             _notNullIfParameterNotNull = _notNullIfParameterNotNull.Add(parameterName);
             SetDataStored();
         }
+
+        private ImmutableArray<int> _interpolatedStringHandlerArguments = ImmutableArray<int>.Empty;
+        public ImmutableArray<int> InterpolatedStringHandlerArguments
+        {
+            get
+            {
+                VerifySealed(expected: true);
+                return _interpolatedStringHandlerArguments;
+            }
+            set
+            {
+                VerifySealed(expected: false);
+                _interpolatedStringHandlerArguments = value;
+                SetDataStored();
+            }
+        }
+
+        private bool _hasInterpolatedStringHandlerArgumentAttributeError = false;
```

Comment:
>private bool _hasInterpolatedStringHandlerArgumentAttributeError = false; [](http://example.com/codeflow?start=8&length=73)

Consider if we can get rid of this separate flag similar to the suggestion for PEParameterSymbol.